circle.yml

```diff
@@ -0,0 +1,2 @@
+general:
+  build_dir: sympy/pysics/tests
```

Comment:
yep
thanks a lot, 
btw I was testing something this should not have come in PR